WebDesignerSamples/WebDesigner_Angular(Core)/ClientApp/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "web-designer-angular",
+  "version": "0.0.0",
+  "scripts": {
+    "deploy-bundles": "ncp ./node_modules/@grapecity/ar-viewer/dist ../wwwroot/assets && ncp ./node_modules/@grapecity/ar-designer/dist ../wwwroot/assets",
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "npm run deploy-bundles && ng build",
+    "build:ssr": "ng run WebDesigner_Angular(Core):server:dev",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "7.0.3",
+    "@angular/common": "7.0.3",
+    "@angular/compiler": "7.0.3",
+    "@angular/core": "7.0.3",
+    "@angular/forms": "7.0.3",
+    "@angular/http": "7.0.3",
+    "@angular/platform-browser": "7.0.3",
+    "@angular/platform-browser-dynamic": "7.0.3",
+    "@angular/platform-server": "7.0.3",
+    "@angular/router": "7.0.3",
+    "@nguniversal/module-map-ngfactory-loader": "^5.0.0-beta.5",
+    "aspnet-prerendering": "^3.0.1",
+    "core-js": "^2.4.1",
+    "rxjs": "^6.3.3",
+    "webpack": "^4.25.1",
+    "zone.js": "^0.8.26"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.10.0",
+    "@angular/cli": "7.0.5",
+    "@angular/compiler-cli": "7.0.3",
+    "@angular/language-service": "7.0.3",
+    "@types/jasmine": "~2.8.3",
+    "@types/jasminewd2": "~2.0.2",
+    "@types/node": "~6.0.60",
+    "codelyzer": "^4.0.1",
+    "jasmine-core": "~2.8.0",
+    "jasmine-spec-reporter": "~4.2.1",
+    "karma": "~2.0.0",
+    "karma-chrome-launcher": "~2.2.0",
+    "karma-coverage-istanbul-reporter": "^1.2.1",
+    "karma-jasmine": "~1.1.0",
+    "karma-jasmine-html-reporter": "^0.2.2",
+    "protractor": "~5.1.2",
+    "ts-node": "~4.1.0",
+    "tslint": "~5.9.1",
+    "typescript": "3.1.6",
+    "webpack-dev-server": "^3.1.10",
+    "@grapecity/ar-viewer": "14.0.1381-beta",
+    "@grapecity/ar-designer": "14.0.602-beta",

```

Comment:
Please use release versions for ar-viewer and ar-designer. Users do not have access to internal versions.

Reply:
Fixed.